aerospike/tests/test_unit.py

```diff
@@ -94,7 +94,7 @@ def test_connection_uses_tls():
 
     with mock.patch('aerospike.client') as client:
         check.get_client()
-        assert client.called_with({'host': check._host, 'tls': tls_config})
+        assert client.assert_called_with({'host': check._host, 'tls': tls_config})
```

Comment:
I think the `assert` is not necessary there:

```suggestion
        client.assert_called_with({'host': check._host, 'tls': tls_config})
```

Reply:
Good catch, I also had to fix the assert because it was wrong: https://github.com/DataDog/integrations-core/blob/master/aerospike/datadog_checks/aerospike/aerospike.py#L307